### DISCUSSION TITLE
Recovering from interrupted jobs ie power outage

### DISCUSSION THREAD

**Discussion by msears101** [automatic-ripping-machine/automatic-ripping-machine#1163] [Q&A]
Hi,

I had a long queue of jobs loaded and I lost power. All the jobs are still there, and not listed on the home screen in the web interface, but seem to not being worked.  I was wondering the correct way to fix this and clean it up.  Also what is the proper way to shutdown the docker container and server if it does have jobs in progress.

 I will probably just restart them all.   

Thanks to the team that created this and maintains it.

**Comment by shitwolfymakes** [comment]
This is unfortunately a side-effect of how ARM is architected at present. I hope to implement failure recovery with the upcoming v3.0 rewrite!
